bin/files.re

```diff
@@ -48,6 +48,22 @@ module Validators = {
     );
 };
 
+module Trusted_validators = {
+  [@deriving yojson]
+  type t = {address: Address.t};
```

Comment:
As discussed we also need to store if it's trusted to be added or removed.

Reply:
I think you've addresses the wrong PR. Update actions (adding/removing) happen on PR #185

Reply:
Yes but this file is designed to be used at #185 so on the current state of the PR this file doesn't move to any meaningful goal.

There is no list of `Trusted_validators`, there is a list of `Trusted_validators_membership_change`

Reply:
> this file doesn't move to any meaningful goal.
It seeds the node-local state

Reply:
Yes but the file is in the wrong format, and it's not created by the setup scripts. 

Add the field containing the kind of membership change.